tests_e2e/orchestrator/lib/agent_test_loader.py

```diff
@@ -55,6 +55,8 @@ class TestSuiteInfo(object):
     images: List[str]
     # The locations (regions) on which the suite must run; if empty, the suite can run on any location
     locations: List[str]
+    # If locations is provided, this determines whether the suite should run on all the suite locations or just one
+    runs_on_all_suite_locations: bool
```

Comment:
I don't feel this parameter is needed, one can already set a specific location in the runbook parameters.

Reply:
+1

Reply:
Removed, and updated combinator to always run on all locations listed in the test suite. Also updated the runbook parameter to allow a comma separated list of locations